packages/@aws-cdk/aws-stepfunctions-tasks/lib/evaluate-expression.ts

```diff
@@ -56,18 +56,17 @@ export class EvaluateExpression implements sfn.IStepFunctionsTask {
   public bind(task: sfn.Task): sfn.StepFunctionsTaskConfig {
     const matches = this.props.expression.match(/\$[.\[][.a-zA-Z[\]0-9]+/g);
 
-    if (!matches) {
-      throw new Error('No paths found in expression');
+    let expressionAttributeValues: {} | undefined;
```

Comment:
The handler does `Object.entries(event.expressionAttributeValues)` and will fail if `undefined`. I don't get why TypeScript doesn't get it here.

Reply:
OK the `as Event´ below is wrong that's why.